examples/chip-tool/BUILD.gn

```diff
@@ -91,6 +91,8 @@ static_library("chip-tool-utils") {
     "commands/icd/ICDCommand.h",
     "commands/pairing/OpenCommissioningWindowCommand.cpp",
     "commands/pairing/OpenCommissioningWindowCommand.h",
+    "commands/pairing/OpenJointCommissioningWindowCommand.cpp",
```

Comment:
Should this be excluded from the build when Joint Fabric is not wanted?

Reply:
This should be excluded from the build in all the cases: it's jf-control-app which is being used for JF and this allows us to keep chip-tool files untouched.

Reply:
This has been excluded from chip-tool in the latest commit.